docs/newdocs/diff_rules_scala.md

```diff
@@ -0,0 +1,25 @@
+### Differences with bazelbuild/rules_scala
+
+* More correctly handles of macros and ijars. See [#445](https://github.com/bazelbuild/rules_scala/issues/445) and [#632](https://github.com/bazelbuild/bazel/issues/632#issuecomment-383318341).
+  * Detect missing `macro = True` attribute. See [#366](https://github.com/bazelbuild/rules_scala/issues/366).
+* More precisely and straightforwardly detects indirect and unused dependencies, via Zinc. See [#235](https://github.com/bazelbuild/rules_scala/issues/235) and [#335](https://github.com/bazelbuild/rules_scala/issues/335).
```

Comment:
I might just remove this line altogether. It will be obsolete if I ever get around to implementing the alternative unused dep checking prototyped in a side branch.